optional/compatible_abc.py

```diff
@@ -9,4 +9,4 @@
 from abc import ABCMeta
 
 
-CompatibleABC = ABCMeta('ABC', (object,), {'__slots__': ()})
+CompatibleABC = ABCMeta("ABC", (object,), {"__slots__": ()})
```

Comment:
I like that it prefers double quotes to single, this has always been my preference.  For some reason I feel it has been historically anti-pythonic though.

Reply:
You can tell black to skip string normalization and it will leave single quoted strings as they are. It prefers doubles as text is more likely to include an apostrophe or single quote than a double quote.